src/proxy-page/steps/addJira.ts

```diff
@@ -11,6 +11,30 @@ export default (config: ConfigService, jiraService: JiraService): Step => {
     const basePath = config.get('web.basePath');
     const version = config.get('version');
 
+    /* fetch Jira issues details and update the title and status for each one */
+    const issuesDetailsPromises = [];
+    $('[data-macro-name="jira"]').each((_, elementJira: cheerio.Element) => {
```

Comment:
I had an error in the example page created. I believe because this selector is not restrictive enough and is taking also the jira-tables which does not have a key. It will be good to debug the error in any case and manage it properly.